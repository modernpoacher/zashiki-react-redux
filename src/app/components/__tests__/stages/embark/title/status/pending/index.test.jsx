import React from 'react'
import snapshotOf, {
  getComponentElement
} from 'react-component-snapshot'

import '@testing-library/jest-dom'

import {
  render
} from '@testing-library/react'

import Pending from '#zashiki-react-redux/app/components/stages/embark/title/status/pending'

describe('#zashiki-react-redux/app/components/stages/embark/title/status/pending', () => {
  describe('Always', () => {
    it('renders', () => {
      expect(snapshotOf(getComponentElement(render(
        <Pending />
      ))))
        .toMatchSnapshot()
    })
  })
})
