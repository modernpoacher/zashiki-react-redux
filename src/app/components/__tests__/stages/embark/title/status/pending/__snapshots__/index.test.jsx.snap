// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#app/components/stages/embark/title/status/pending Always renders 1`] = `
<header
  className="embark pending"
/>
`;
